package.json

```diff
@@ -44,11 +48,17 @@
     "@types/isomorphic-fetch": "^0.0.36",
     "@types/jest": "^29.4.0",
     "@types/node": "^18.14.0",
+    "@typescript-eslint/eslint-plugin": "^5.54.1",
+    "@typescript-eslint/parser": "^5.54.1",
     "babel-jest": "^29.4.3",
     "esbuild": "^0.17.10",
+    "eslint": "^8.35.0",
+    "eslint-config-prettier": "^8.7.0",
+    "eslint-plugin-prettier": "^4.2.1",
     "jest": "^29.4.3",
-    "prettier": "^2.1.1",
+    "prettier": "^2.8.4",
     "ts-jest": "^29.0.5",
+    "ts-node": "^10.9.1",
```

Comment:
What is the `ts-node` package needed for?

Reply:
Will verify if it's needed or not.

Reply:
Don't believe it is. Bye 👋